environment.yml

```
@@ -3,6 +3,7 @@ channels:
 
 dependencies:
   - jupyter_contrib_nbextensions==0.5.1
+  - jupyterhub==2.3.1
```

For user images, the conda package to install is juoyterhub-base

I rhink a pin on patch level is too tight as well. Maybe less than 3 or larger than 2.3.1 ?

---

[The docs list 3 pkgs](https://github.com/conda-forge/jupyterhub-feedstock#installing-jupyterhub-feedstock): jupyterhub, jupyterhub-base, jupyterhub-singleuser. I suggest pinning the two that are already present?

```suggestion
  - jupyterhub-base==2.3.1
  - jupyterhub-singleuser=2.3.1
```

> I rhink a pin on patch level is too tight as well.

I figured we should use the same patch version we're using everywhere else, since we've tested it and we know it works? Do you feel strongly about pinning this strict?